docs/Getting-Started.md

```diff
@@ -20,7 +20,63 @@ $ react-native link react-native-webview
 
 _NOTE: If you ever need to uninstall React Native WebView, run `react-native unlink react-native-webview` to unlink it._
 
-#### 3. Import the webview into your component
+#### 3. Add support for File Upload
```

Comment:
This is a really valuable section! In my opinion, this section works better in the `Guide.md`. I like having the Getting Started cover the most basic use case (which wouldn't include file upload). Then the "In-Depth Guide" has more specific use-cases like this one.

Reply:
Makes sense. Will update the docs.